ngmix/priors/__init__.py

```diff
@@ -0,0 +1,3 @@
+# flake8: noqa
+from .priors import *
+from .random import make_rng, srandu
```

Comment:
It would be better to remove srandu entirely and instead make the functions that call it *require* having an rng sent.  This stems from the times when I used the default rng from numpy